src/containers/Pager.js

```diff
@@ -19,16 +20,24 @@ const mapDispatchToProps = dispatch => bindActionCreators({
   refreshData: actions.refreshData
 }, dispatch)
 
-const handlers = {
+const changePageHandlers = {
   changePage: props => event => {
     const nextPage = event.target.getAttribute('data-page')
     props.refreshData(props.name, {...props.filter, page: nextPage})
   }
 }
 
+const mapNumbersPagesHandlers = {
+  mapNumberedPagerButtons: props => (page, index) =>
+    <PagerButton loading={props.loading} key={index} page={page} currentPage={props.filter.page} changePage={props.changePage}>
+      {page + 1}
+    </PagerButton>
+}
+
 const enhance = compose(
   connect(mapStateToProps, mapDispatchToProps),
-  withHandlers(handlers)
+  withHandlers(changePageHandlers),
+  withHandlers(mapNumbersPagesHandlers)
```

Comment:
I believe we should only need to wrap this once with `withHandlers` since it takes in an object

Reply:
can't do that, the second function needs access to the first handler with props in it.